src/Components/AddLodgingInfo/AddLodgingInfo.js

```diff
@@ -1,96 +1,123 @@
-import React, { Component } from 'react';
-import Header from '../Header/Header'
-import Footer from '../Footer/Footer'
-import { 
-    StyleSheet, 
-    Text, 
-    View, 
-    ScrollView,
-    TouchableOpacity,
-    Keyboard,
-    TextInput 
-  } from 'react-native';
+import React, { Component } from "react";
+import DatePicker from "react-native-datepicker";
+import Header from "../Header/Header";
+import Footer from "../Footer/Footer";
+import {
+  StyleSheet,
+  Text,
+  View,
+  ScrollView,
+  TouchableOpacity,
+  Keyboard,
+  TextInput
+} from "react-native";
 
 export default class AddLodgingInfo extends Component {
   constructor() {
-    super()
+    super();
     this.state = {
-      countryLodge: '',
-      cityLodge: '',
-      lodgeName: '',
-      startLodge: '',
-      endLodge: ''
-    }
-  }
-
-  handleInputs = () => {
-    this.setState({ [e.target.name] : e.target.value })
```

Comment:
Is this working? I think we still need to use the name of the state's property we want to update and the e.nativeEvent.text